mock-echo-app/src/cmr/mock_echo/api/urs.clj

```diff
@@ -1,15 +1,18 @@
 (ns cmr.mock-echo.api.urs
   "Defines routes for mocking URS"
-  (:require [compojure.core :refer :all]
-            [cheshire.core :as json]
-            [clojure.data.xml :as x]
-            [cmr.common.xml :as cx]
-            [cmr.common.mime-types :as mt]
-            [clojure.string :as str]
-            [cmr.common.services.errors :as errors]
-            [cmr.mock-echo.data.urs-db :as urs-db]
-            [clojure.data.codec.base64 :as b64]
-            [ring.util.response :as rsp]))
+  (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as b64]
+   [clojure.data.xml :as x]
+   [clojure.string :as str]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.services.errors :as errors]
+   [cmr.common.xml :as cx]
+   [cmr.mock-echo.data.urs-db :as urs-db]
+   [cmr.transmit.config :as transmit-config]
+   [compojure.core :refer :all]
+   [ring.util.response :as rsp]))
 
```

Comment:
Ah, good to know. 

Hrm, I wonder why he's treating `:require` like an element of a data structure instead of a (special) form?

/me looks at source ...

Ah, probably because the `ns` macro treats the next group of forms after the docstring as a data structure (assigned to 'attr-map?', in particular). That answers that question!

Thanks for the link!